sklearn/metrics/_plot/tests/test_common_curve_display.py

```diff
@@ -176,9 +174,9 @@ def test_display_curve_estimator_name_multiple_calls(
         ),
     ],
 )
-@pytest.mark.parametrize(
-    "Display", [DetCurveDisplay, PrecisionRecallDisplay, RocCurveDisplay]
-)
+# Add separate test for displays that have converted to names?,
+# add note to remove this one in 1.9
+@pytest.mark.parametrize("Display", [DetCurveDisplay, PrecisionRecallDisplay])
```

Comment:
I was thinking that I could add a separate test that uses `names` (instead of `estimator_name`), so we can delete this test once we've switched all these estimators across to `names`?
This avoids complicate if/else inside this test.
WDYT?